### ISSUE TITLE
Webui does not start

### ISSUE THREAD

**Issue by Pandananana** [oobabooga/text-generation-webui#4240]
### Describe the bug

I am on a VPN to a remote linux server and have accessed it via SSH. I have modified the CMD_FLAGS.txt file to include

--share --gradio-auth user:pswd

When i run the file ./start_linux.sh I get this output:

```
~/projects/text-generation-webui$ ./start_linux.sh

    WARNING: You haven't downloaded any model yet.
    Once the web UI launches, head over to the "Model" tab and download one.

2023-10-05 22:32:03 WARNING:The gradio "share link" feature uses a proprietary executable to create a reverse tunnel. Use it with care.
```

After stopping with Control + C, I get this extra info:

```
2023-10-09 15:49:54 WARNING:The gradio "share link" feature uses a proprietary executable to create a reverse tunnel. Use it with care.
^CTraceback (most recent call last):
  File "/home/awe/projects/text-generation-webui/one_click.py", line 361, in <module>
    launch_webui()
  File "/home/awe/projects/text-generation-webui/one_click.py", line 321, in launch_webui
    run_cmd(f"python server.py {flags}", environment=True)
  File "/home/awe/projects/text-generation-webui/one_click.py", line 136, in run_cmd
    result = subprocess.run(cmd, shell=True, capture_output=capture_output, env=env)
  File "/home/awe/projects/text-generation-webui/installer_files/env/lib/python3.10/subprocess.py", line 505, in run
    stdout, stderr = process.communicate(input, timeout=timeout)
  File "/home/awe/projects/text-generation-webui/installer_files/env/lib/python3.10/subprocess.py", line 1146, in communicate
    self.wait()
  File "/home/awe/projects/text-generation-webui/installer_files/env/lib/python3.10/subprocess.py", line 1209, in wait
    return self._wait(timeout=timeout)
  File "/home/awe/projects/text-generation-webui/installer_files/env/lib/python3.10/subprocess.py", line 1959, in _wait
    (pid, sts) = self._try_wait(0)
  File "/home/awe/projects/text-generation-webui/installer_files/env/lib/python3.10/subprocess.py", line 1917, in _try_wait
    (pid, sts) = os.waitpid(self.pid, wait_flags)
KeyboardInterrupt

```

The program doesn't print anything else and just hangs. It doesn't print the Gradio URL so I can visit the Web UI. I have tried modifying the server.py file, but nothing happens. I have also tried adding a print("Test") multiple places in the file, but still nothing is printed to the terminal. When I stop the program with Control + C, it shuts down, but when running Top, i can see that there is a python process that is still running, as if it hasn't shut down correctly?

### Is there an existing issue for this?

- [X] I have searched the existing issues

### Reproduction

Run ./start_linux.sh

### Logs

```shell
/projects/text-generation-webui$ ./start_linux.sh 


*******************************************************************
* WARNING: You haven't downloaded any model yet.
* Once the web UI launches, head over to the "Model" tab and download one.
*******************************************************************


2023-10-09 15:49:54 WARNING:The gradio "share link" feature uses a proprietary executable to create a reverse tunnel. Use it with care.
^CTraceback (most recent call last):
  File "/home/awe/projects/text-generation-webui/one_click.py", line 361, in <module>
    launch_webui()
  File "/home/awe/projects/text-generation-webui/one_click.py", line 321, in launch_webui
    run_cmd(f"python server.py {flags}", environment=True)
  File "/home/awe/projects/text-generation-webui/one_click.py", line 136, in run_cmd
    result = subprocess.run(cmd, shell=True, capture_output=capture_output, env=env)
  File "/home/awe/projects/text-generation-webui/installer_files/env/lib/python3.10/subprocess.py", line 505, in run
    stdout, stderr = process.communicate(input, timeout=timeout)
  File "/home/awe/projects/text-generation-webui/installer_files/env/lib/python3.10/subprocess.py", line 1146, in communicate
    self.wait()
  File "/home/awe/projects/text-generation-webui/installer_files/env/lib/python3.10/subprocess.py", line 1209, in wait
    return self._wait(timeout=timeout)
  File "/home/awe/projects/text-generation-webui/installer_files/env/lib/python3.10/subprocess.py", line 1959, in _wait
    (pid, sts) = self._try_wait(0)
  File "/home/awe/projects/text-generation-webui/installer_files/env/lib/python3.10/subprocess.py", line 1917, in _try_wait
    (pid, sts) = os.waitpid(self.pid, wait_flags)
KeyboardInterrupt
```

### System Info

```shell
Linux 5.15.0-79-generic #86-Ubuntu SMP Mon Jul 10 16:07:21 UTC 2023 x86_64 x86_64 x86_64 GNU/Linux

Intel(R) Core(TM) i5-10600K CPU @ 4.10GHz

NVIDIA 4060Ti 16GB
```

**Comment by Pandananana** [comment]
Thanks for the quick response! Sadly --auto-launch does not seem to make a difference.

I also noticed that running the nvidia-smi command does not respond. Instead it just hogs 100% of the CPU. I have tried looking for solutions online but can't find the same problem (others describe nvidia-smi freezing the system or not being able to be killed, which isn't an issue here). Is it possible that the text-generation-webui installation caused something to break with regards to the GPU settings?
